src/utilities/util_dump.c

```diff
@@ -652,18 +661,60 @@ dump_record(WT_CURSOR *cursor, bool reverse, bool json)
         infix = "\n";
         suffix = "\n";
     }
-    while ((ret = (reverse ? cursor->prev(cursor) : cursor->next(cursor))) == 0) {
-        if ((ret = cursor->get_key(cursor, &key)) != 0)
-            return (util_cerr(cursor, "get_key", ret));
-        if ((ret = cursor->get_value(cursor, &value)) != 0)
-            return (util_cerr(cursor, "get_value", ret));
-        if (fprintf(
-              fp, "%s%s%s%s%s%s", json && once ? "," : "", prefix, key, infix, value, suffix) < 0)
-            return (util_err(session, EIO, NULL));
-        once = true;
+
+    while (ret == 0) {
```

Comment:
Should `ret` be initialized before comparison?

Reply:
It is initialized by `WT_DECL_RET`:
```
#define WT_DECL_RET int ret = 0
```

Reply:
Ah got it.